### ISSUE TITLE
Vis/Essentia Locomotive; Integration with Thaumcraft

### ISSUE THREAD

**Issue by Okkasionen** [Railcraft/Railcraft#723]
I watched CovertJaguars Darktower series again for some creative input for my own Railcraft, Buildcraft and Forestry centered Singleplayer world. In the last Episodes he stated that he was working on an essentia-driven locomotive. But as its seems nothing has came out of these plans. I have to say it would be awesome! Please add it!

**Comment by liach** [comment]
Yes the implementation exists [here] (https://github.com/CovertJaguar/Railcraft/blob/master/src/main/java/mods/railcraft/common/carts/EntityLocomotiveSteamMagic.java) (feels weird)

**Comment by CovertJaguar** [comment]
It's largely untested (which means most likely completely broken) and there is no GUI for it. I wasn't ever quite sure what the GUI should look like.

**Comment by liach** [comment]
Closing since thaumcraft is unclear in its future.

**Comment by liach** [comment]
Tc is back.

**Comment by liach** [comment]
https://www.patreon.com/posts/sad-farewell-23967282

**Comment by Voxel-Friend** [comment]
TC is not dead. It is in good hands

**Comment by liach** [comment]
I suggest you coming to Railcraft discord and talk about this question of thaumcraft in the general channel.
